torch/csrc/jit/passes/onnx/constant_fold.cpp

```diff
@@ -638,14 +647,26 @@ void ConstantFoldONNX(Block* b, ParamMap& paramsDict, int opset_version) {
       // Constant folding is not supported for this op. Skip it.
       continue;
     }
-    // Create a new input to the block (prim::Param node output). Add a
-    // corresponding entry in valToParamMap. Replace the downstream inputs
-    // with this value, and disconnect all the input values of the folded node.
+
     at::Tensor updatedVal = *updatedValWrapped;
-    auto newSourceNodeOutput = b->addInput();
-    valsToParamsMap.insert(
-        {newSourceNodeOutput,
-         std::make_pair(newSourceNodeOutput->debugName(), updatedVal)});
+    auto newSourceNodeOutput = [&]() -> Value* {
+      if (onnx_constant_fold::hasParamInput(node, valsToParamsMap)) {
```

Comment:
can we omit the namespace when calling the function?

Reply:
Couldn't without more changes, the caller function is outside of `onnx_constant_fold` namespace.